src/ar_http_iface_cowboy_handler.erl

```diff
@@ -0,0 +1,1142 @@
+-module(ar_http_iface_cowboy_handler).
+-behaviour(cowboy_handler).
+-export([init/2]).
+-include("ar.hrl").
+
+%%%===================================================================
+%%% Cowboy handler callbacks.
+%%%===================================================================
+
+init(Req0, State) ->
+	{Status, Headers, Body} = do_handle(Req0),
+	CowboyStatus = handle208(Status),
+	Req1 = cowboy_req:reply(CowboyStatus, Headers, Body, Req0),
+	{ok, Req1, State}.
+
+%%%===================================================================
+%%% Private functions.
+%%%===================================================================
+
+do_handle(Req) ->
+	%% Inform ar_bridge about new peer, performance rec will be updated from ar_metrics
```

Comment:
The ar_metrics mention is irrelevant now.

Reply:
I'm not sure I understand the comment... Should I say the following?
> performance rec will be updated from cowboy_metrics_h